utils/webassembly/build-mac.sh

```diff
@@ -12,6 +12,7 @@ $SWIFT_PATH/utils/build-script --wasm \
     -DSWIFT_BUILD_SOURCEKIT=FALSE \
     -DSWIFT_ENABLE_SOURCEKIT_TESTS=FALSE \
     -DSWIFT_BUILD_SYNTAXPARSERLIB=FALSE \
+    -DSWIFT_INCLUDE_TESTS=FALSE \
```

Comment:
Wouldn't this disable all the tests on macOS completely? Or is there no way to keep the tests, but just to avoid building Block for wasm?

Reply:
Oops, I didn't intended to disable all the tests. I reverted this and disable only BlocksRuntime